package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "story-book",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
I can see som dev dependencies - please move them...

Reply:
remone them or move them to dependencies

Reply:
I am not sure what you have meant with you comment response. What are you going to do? Are you suggesting something?

Reply:
What should i do with dev-dep? remove them or make them only dependencies

Reply:
dependencies that are needed in the browser should be placed in `dependencies`

dev dependencies (like jest) are not needed in the browser and should be placed in `devDependencies`